### PR TITLE
Ps 186 bug submit error

### PR DESCRIPTION
**Pull request by Tibor22**
Fixed error handling on client site and added some styling to comments

### CODE REVIEW

**Comment by d-username** [comment]
ISSUE Number: 186
https://github.com/boolean-uk/team-dev-client/issues/186

**Review by shahzadnaeem — DISMISSED** [review]
Looks ok. Informed that Ed already mentioned validation should be client side but has accepted server side in this case.

File: src/components/posts/Comment.js [hunk 1]
```diff
@@ -3,14 +3,18 @@ import { formatDate } from '../../Helper/dateFormatter';
...
 	return (
 		<li>
-			<div className='name-time-container'>
-				<div className='name'>
-					{comment.profile.firstName + ' '}
-					{comment.profile.lastName}
+			<div className='comment-container'>
+				<div className='name-time-container'>
+					<div className='name'>
+						{comment.profile.firstName + ' '}
```
Comment (Auenc) [inline src/components/posts/Comment.js:9]: Did you try to get the space in-between the first name and last name by adding a space in the JSX?

Something like: 
{comment.profile.firstName} {comment.profile.lastName}
```js

```

File: src/components/posts/style.css [hunk 2]
```diff
@@ -115,3 +128,24 @@
...
+	flex-direction: column;
+	align-items: center;
+}
+
+.error {
+	color: #d8000c;
+	background-color: #ffbaba;
+	background-image: url('https://i.imgur.com/GnyDvKN.png');
+	background-repeat: no-repeat;
+	/* background-position: center; */
```
Comment (Auenc) [inline src/components/posts/style.css:143]: let's remove this commented css

**Review by Hoolio333 — DISMISSED** [review]

**Review by Auenc — DISMISSED** [review]
LGTM

**Review by Auenc — APPROVED** [review]
LGTM

**Review by d-username — APPROVED** [review]

**Pull request merged**